Gems/Atom/RHI/DX12/Code/Source/Platform/Windows/RHI/Device_Windows.cpp

```diff
@@ -886,6 +888,12 @@ namespace AZ
 
         RHI::ResultCode Device::BeginFrameInternal()
         {
+            static uint32_t frameIndex = 0;
```

Comment:
does this need to be a  64 bit int?

Reply:
no, it's just a useful sequential value that we pass to D3D12MA via [SetCurrentFrameIndex(UINT).](https://gpuopen-librariesandsdks.github.io/D3D12MemoryAllocator/html/class_d3_d12_m_a_1_1_allocator.html#a468ba0c93121eaaee402b08775f1dd11)